blackberggroup/VA-National-Centers
# Feature/rp 6170246451/drhci landing page

## What type of PR is this? (check all applicable)


## Description
Coded DRHCI landing page for layout review.

Pending items to add for review:

CTA links:
- Explore Innovations
- Connect Now

News article:
- Missing news carousel article content

## Related Tickets & Documents
- Monday Task # 6170246451
https://blackberg-group.monday.com/boards/1931719392/pulses/6170246451

## QA Instructions, Screenshots, Recordings

Check for available content, UI layout from figma from widescreen to mobile. This has been test in Chrome emulator and will test on actual Samsung device upon launch to dev-innovation.

Also spot checked NCCHI pages as they are incuded in this repo. 

Figma link: https://www.figma.com/file/sAThM2RYd5horvGZK99iNb/DRHCI---Digital-and-Remote-Health-Center?node-id=1%3A2&mode=dev

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/nci2i/1_masthead.html
 - _includes/nci2i/2_mission.html
 - _includes/nci2i/3_news.html
 - _includes/nci2i/4_director.html
 - _includes/nci2i/5_connect.html
 - _site/assets/img/nci2i/bubble-lines-sm.svg
 - _site/assets/img/nci2i/bubble-lines.svg
 - _site/assets/img/nci2i/lindsay-reigler.jpg
 - _site/assets/img/nci2i/logo.svg
 - _site/assets/img/nci2i/masthead-image.jpg
 - _site/assets/img/nci2i/masthead-img.jpg
 - _site/assets/img/nci2i/news/nci2i-news-feet.jpg
 - _site/assets/img/nci2i/news/nci2i-news-feetpad.jpg
 - _site/assets/img/nci2i/news/nci2i-news-helpp.png
 - _site/assets/img/nci2i/news/nci2i-news-telehealth.png
 - _site/assets/img/nci2i/news/nci2i-news-toes.jpg
 - _site/assets/img/nci2i/news/nci2i-news-walking.jpg
 - _site/assets/img/nci2i/vha-nci2i-logo.svg
 - _site/assets/js/nci2i-home-news.js
 - _site/assets/js/nci2i-news.js
 - _site/assets/sass/vha/pages/home/connect/_index.scss
 - _site/assets/sass/vha/pages/home/director/_index.scss
 - _site/assets/sass/vha/pages/home/masthead/_index.scss
 - _site/assets/sass/vha/pages/home/mission/_index.scss
 - _site/assets/sass/vha/pages/home/news/_index.scss
 - assets/img/nci2i/bubble-lines-sm.svg
 - assets/img/nci2i/bubble-lines.svg
 - assets/img/nci2i/lindsay-reigler.jpg
 - assets/img/nci2i/logo.svg
 - assets/img/nci2i/masthead-image.jpg
 - assets/img/nci2i/masthead-img.jpg
 - assets/img/nci2i/news/nci2i-news-feet.jpg
 - assets/img/nci2i/news/nci2i-news-feetpad.jpg
 - assets/img/nci2i/news/nci2i-news-helpp.png
 - assets/img/nci2i/news/nci2i-news-telehealth.png
 - assets/img/nci2i/news/nci2i-news-toes.jpg
 - assets/img/nci2i/news/nci2i-news-walking.jpg
 - assets/img/nci2i/vha-nci2i-logo.svg
 - assets/js/nci2i-home-news.js
 - assets/sass/vha/pages/home/connect/_index.scss
 - assets/sass/vha/pages/home/director/_index.scss
 - assets/sass/vha/pages/home/masthead/_index.scss
 - assets/sass/vha/pages/home/mission/_index.scss
 - assets/sass/vha/pages/home/news/_index.scss
 - _includes/footer.html
 - _site/assets/css/styles.css
 - _site/assets/img/ncchi/team-arreola.png
 - _site/assets/img/ncchi/team-blumke.png
 - _site/assets/img/ncchi/team-higgins.png
 - _site/assets/img/ncchi/team-hoover.png
 - _site/assets/img/ncchi/team-osborne.png
 - _site/assets/img/ncchi/team-pham.png
 - _site/assets/img/ncchi/team-salatin.png
 - _site/assets/img/ncchi/team-veigulis.png
 - _site/assets/img/ncchi/team-vrublevskiy.png
 - _site/assets/img/ncchi/team-ware.png
 - _site/assets/js/functions.js
 - _site/assets/sass/vha/pages/home/_index.scss
 - _site/index.html
 - assets/css/styles.css
 - assets/js/functions.js
 - assets/sass/vha/pages/home/_index.scss
 - index.html
